mapdata/WarcraftLegacies/BuffData/Skin/1211379778.json

```
@@ -0,0 +1,29 @@
+{
+  "OldId": 1835619650,
+  "NewId": 1211379778,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1885959270,
+      "Type": 3,
+      "Value": "Stunned"
+    },
+    {
+      "Id": 1700951398,
+      "Type": 3,
+      "Value": "This unit has been stunned. It cannot move, attack or cast spells."
```

Commas don't work properly in custom buffs so I have to remove it.
```suggestion
      "Value": "This unit has been stunned. It cannot move or attack or cast spells."
```